### PR TITLE
PORTALS-2690 - OrientationBanner for Cohort Builder

### PR DESCRIPTION
**Pull request by nickgros**
- Add OrientationBanner to cohort builder page in EL Portal
- Fix issue where the nav menu would block content at small (but not xs) screen sizes
- Replace undesirable whitespace (margin) with padding on explore pages header at small screen sizes

OrientationBanner:

image

## CSS changes

|  |Before|After|
|---|---|---|
|Whitespace on Explore header, xs size| image | image |
|Nav menu content cutoff on small size| image | image |

### CODE REVIEW

File: apps/portals/src/_ExploreMenu.scss [hunk 1]
```diff
@@ -36,7 +36,7 @@
     .title {
       font-size: 14px;
       text-align: center;
-      margin: 1rem;
```
Comment (nickgros) [inline apps/portals/src/_ExploreMenu.scss:39]: Has a background color, so padding looks better than margin

File: apps/portals/src/_Navbar.scss [hunk 2]
```diff
@@ -24,9 +24,6 @@
   $mb-menu-item-padding: 46px;
   $mb-menu-divider: 1px solid #b2b3b6;
 
-  .nav + .spacer {
-    display: none;
-  }
```
Comment (nickgros) [inline apps/portals/src/_Navbar.scss:29]: Always display the spacer.

File: apps/portals/src/configurations/elportal/routesConfig.tsx
Comment (nickgros) [inline apps/portals/src/configurations/elportal/routesConfig.tsx:1]: Converted to `tsx` to easily add markup to the OrientationBanner text as in the design

File: apps/portals/src/configurations/elportal/routesConfig.tsx [hunk 3]
```diff
@@ -208,6 +209,31 @@ const routes: GenericRoute[] = [
...
+              name: 'CohortBuilder',
+              title: 'Getting Started With Explore Data by Participants',
+              text: (
+                <>
+                  This page helps you find data related to anonymized
+                  Participants. Add filters to build a list of participants. If
+                  you have a list of participant IDs, click{' '}
+                  <strong>Filter by Participant</strong> to add them. Once
+                  you’ve built your cohort, you can download the related data
+                  files, <strong>Send to Cavatica</strong> for analysis, or just{' '}
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/elportal/routesConfig.tsx:225]: Change capitalization to "Send to CAVATICA" per Milan

**Comment by nickgros** [comment]
Merging since copy has been fixed

**Pull request merged**